unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -78,7 +83,7 @@ export class CreatorLock extends React.Component {
         )}
         {!showEmbedCode &&
           showKeys && (
-          <LockPanel>
+          <LockPanel onClick={this.preventPropagation}>
```

Comment:
Instead of using preventPropagation and binding it in the constructor, I recommend:

```js
<LockPanel onClick={e => e.preventPropagation}>
```

This is much simpler and achieves the same result